<launch>
    <node pkg="state_estimation" type="data_format.py" name="data_collection" respawn="false" output="screen">
        <param name="update_rate" value="100" />
        <param name="radius_earth" value="6378.1370" />
        <param name="laditude_offset" value="0" />
        <param name="longitude_offset" value="0" />
        <param name="frame_rate" value="10" />
        <param name="output_dir" value="$(find state_estimation)/data" />
    </node>
</launch>